exemplar/src/main/kotlin/com/squareup/exemplar/HelloWebAction.kt

```diff
@@ -13,8 +10,11 @@ import javax.inject.Singleton
 class HelloWebAction : WebAction {
     @Get("/hello/{name}")
     @ResponseContentType(MediaTypes.APPLICATION_JSON)
-    fun hello(@PathParam name: String, @RequestHeaders headers: Headers): HelloResponse {
-        return HelloResponse("YO", name.toUpperCase())
+    fun hello(@PathParam name: String, @RequestHeaders headers: Headers,
+            @QueryParam("nickname") nickName: String?,
+            @QueryParam("greetings") greetings: List<String>?): HelloResponse {
```

Comment:
Technically `"greetings"` isn't required in this case, correct (it should deduce the query param name from the name of the parameter)? If so, plz remove the unnecessary qualifier (and do the same with `"nickname"` above). If not, plz make it so that we deduce the query name from the name of the parameter a la `@PathParam` :)

Reply:
Yup, optional, will dump